server/src/services/organizations.service.ts

```diff
@@ -318,7 +318,7 @@ export class OrganizationsService {
     const userParams = <User>{
       firstName: inviteNewUserDto.first_name,
       lastName: inviteNewUserDto.last_name,
-      email: inviteNewUserDto.email,
+      email: inviteNewUserDto.email ? inviteNewUserDto.email.toLowerCase() : '',
```

Comment:
```suggestion
      email: inviteNewUserDto.email?.toLowerCase(),
```

Reply:
please check if we can do it in dto transformations

Reply:
It is Possible